integration/presto/README.md

```diff
@@ -20,14 +20,10 @@
 Please follow the below steps to query carbondata in presto
 
 ### Config presto server
-* Download presto server 0.166 : https://repo1.maven.org/maven2/com/facebook/presto/presto-server/
-* Finish configuration as per https://prestodb.io/docs/current/installation/deployment.html
-  for example:
+* Download presto server ( >= 0.166) : https://repo1.maven.org/maven2/com/facebook/presto/presto-server/
```

Comment:
I tried using presto 0.179 with carbondata 1.2.0, it looks fine. But I am going to change `>=` to `=`.